File: Assignment_1/C.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class C {

    public static void main(String []args) throws IOException{
        InputStreamReader isr = new InputStreamReader(System.in);
        BufferedReader br = new BufferedReader(isr);
        String inp_1 = br.readLine();
        int k = Integer.parseInt(inp_1.split(" ")[1]);

        String inp_2 = br.readLine();
        String[] array = inp_2.split(" ");

        StringBuilder result = new StringBuilder("");

        for(int i = k-1; i >= 0; i --){
            result.append(array[i] + " ");
        }
        System.out.println(result);


    }
 }

File: Assignment_1/C.py
inp_1 = input()
inp_2 = input()

arr_len, k = list(map(int, inp_1.split()))
arr = list(map(int,inp_2.split()))


for i in range(k- 1,-1, -1):
    print(arr[i], end = ' ')

print()